src/styles.scss

```diff
@@ -1 +1,2 @@
 /* You can add global styles to this file, and also import other style files */
+@import "primeicons/primeicons.css";
```

Comment:
Remove from here and add it to the angular.json 

Around line 59 in angular.json : "node_modules/primeicons/primeicons.css"